src/clj/puppetlabs/services/legacy_routes/legacy_routes_service.clj

```diff
@@ -27,7 +28,9 @@
           master-route-config (master-core/get-master-route-config
                                 master-ns
                                 config)
-          master-route-handler (-> (master-core/root-routes handle-request)
+          jruby-service (tk-services/get-service this :JRubyPuppetService)
+          master-route-handler (-> (master-core/root-routes handle-request
+                                                            jruby-service)
```

Comment:
Yuck :(  The legacy routes never need to use the jruby for anything, right?  Not sure I can think of a better way to do this, nor whether or not it's worth putting much thought into, but it does make me a little :cry: